src/test/scala/com/codacy/client/bitbucket/v2/UserSpecs.scala

```diff
@@ -1,195 +1,30 @@
-                  |},
-                  | "team": {
-                  |   "username": "testteam1",
-                  |   "display_name": "TestTeam1",
-                  |   "type": "team",
-                  |   "uuid": "{85ea8027-2a7f-4094-828d-f20c935d373a}",
-                  |   "links": {
-                  |     "self": {
-                  |       "href": "https://bitbucket.org/!api/2.0/teams/testteam1"
-                  |     },
-                  |     "html": {
-                  |       "href": "https://bitbucket.org/testteam1/"
-                  |     },
-                  |     "avatar": {
-                  |       "href": "https://bitbucket.org/account/testteam1/avatar/"
-                  |     }
-                  |   }
-                  | }
-                  |},
-                  |{
-                  | "permission": "collaborator",
-                  | "type": "team_permission",
-                  | "user": {
-                  |   "username": "jllopes",
-                  |   "display_name": "João Lopes",
-                  |   "account_id": "123asdfas87afsd8f9asd7as",
-                  |   "links": {
-                  |     "self": {
-                  |       "href": "https://bitbucket.org/!api/2.0/users/jllopes"
-                  |     },
-                  |     "html": {
-                  |       "href": "https://bitbucket.org/jllopes/"
-                  |     },
-                  |     "avatar": {
-                  |       "href": "https://bitbucket.org/account/jllopes/avatar/"
-                  |     }
-                  |   },
-                  | "nickname": "jllopes",
-                  | "type": "user",
-                  | "uuid": "{42417371-64fe-4cde-b528-b7454e8a0aaf}"
-                  |},
-                  | "team": {
-                  |   "username": "testteam2",
-                  |   "display_name": "TestTeam2",
-                  |   "type": "team",
-                  |   "uuid": "{7ec1171b-3df1-4c00-bfcf-2d4eda99e44a}",
-                  |   "links": {
-                  |     "self": {
-                  |       "href": "https://bitbucket.org/!api/2.0/teams/testteam2"
-                  |     },
-                  |     "html": {
-                  |       "href": "https://bitbucket.org/testteam2/"
-                  |     },
-                  |     "avatar": {
-                  |       "href": "https://bitbucket.org/account/testteam2/avatar/"
-                  |     }
-                  |   }
-                  | }
-                  |}
-                  |]
-                """.stripMargin
-    val json = Json.parse(input)
+    val rawJson = TestUtils.getTestContent("src/test/resources/user-service/user_permissions_teams.json");
+    val json = Json.parse(rawJson)
     val value = json.validate[Seq[TeamWithPermission]]
 
-    value.fold(e => fail(s"$e"), teams => teams.length shouldBe 2)
+    value.fold(e => fail(s"$e"), teams => teams.length shouldBe 1)
```

Comment:
Why it is only 1 now? Can we keep testing a use case with 2?

Reply:
Because I can't generate data with 2, and I don't feel like it changes the concept of this test